Add `platform prerequisites` for ambient, and simplify `getting started` ambient guide (#14709)
istio/istio.io: content/en/docs/ops/ambient/getting-started/index.md

{{< boilerplate ambient-alpha-warning >}}

This guide lets you quickly evaluate Istio's {{< gloss "ambient" >}}ambient mode{{< /gloss >}}. These steps require you to have a {{< gloss >}}cluster{{< /gloss >}} running a

> [supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can install Istio ambient mode on [any supported Kubernetes platform](/docs/setup/platform-setup/), but this guide will assume the use of [kind](https://kind.sigs.k8s.io/) simplicity.

Reviewer: ```suggestion
[supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can install Istio ambient mode on [any supported Kubernetes platform](/docs/setup/platform-setup/), but this guide will assume the use of [kind](https://kind.sigs.k8s.io/) for simplicity.
```

> [supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can install Istio ambient mode on [any supported Kubernetes platform](/docs/setup/platform-setup/), but this guide will assume the use of [kind](https://kind.sigs.k8s.io/) for simplicity.

Reviewer 2: Long line, recommend breaking them for ease of review.

```suggestion
[supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}).
You can install Istio ambient mode on [any supported Kubernetes platform](/docs/setup/platform-setup/), but this guide will assume the use of [kind](https://kind.sigs.k8s.io/) for simplicity.
```

## Download and install {#download}

1.  Install [kind](https://kind.sigs.k8s.io/)

1.  Download the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.

> 1.  For this guide, we will deploy a new local `kind` cluster with the following command:

Reviewer: Guides should be in the second person: "You will X"

Author: Ugh, bad habit.

## Download and install {#download}

1.  Install [kind](https://kind.sigs.k8s.io/)

1.  Download the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.

> 1.  For this guide, you will deploy a new local `kind` cluster with the following command:

Reviewer: I reckon just

```suggestion
1.  Deploy a new local `kind` cluster:
```

1.  Download the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.

> 1.  For this guide, we will deploy a new local `kind` cluster with the following command:

Reviewer: ```suggestion
1.  Deploy a new `kind` cluster locally with the following command:
```

assumes you have Docker installed but doesn't state it?  (Didn't before)

Author: The first step is a `kind` command (after we tell you in the preamble to use `minikube`), so it doesn't really change the assumption from the prev. version.

For this particular quickstart guide, we are having you run `kind` commands, which will require you to install `kind`, which will require you to install `Docker`. I can be more explicit tho.

 If you want to deviate from the "from nothing" local quickstart, just following `content/en/docs/ops/ambient/install/helm-installation/index.md` is probably better.
